### PR TITLE
Migrated scrolling.md to null safety

### PR DESCRIPTION
**Pull request by miquelbeltran**
cc. @sfshaza2 @RedBrogdon @domesticmouse

### CODE REVIEW

File: null_safety_examples/cookbook/testing/integration/scrolling/pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: scrolling
+description: A new Flutter project.
+
+environment:
+  sdk: '>=2.12.0 <3.0.0'
```
Comment (domesticmouse) [inline null_safety_examples/cookbook/testing/integration/scrolling/pubspec.yaml:5]: Fun question going forward will be when to uplift the minimum to 2.13.0 and the like. I'm guessing this sample can't take advantage of the type aliases so 2.12.0 is fine as is.

File: null_safety_examples/cookbook/testing/integration/scrolling/test_driver/app.dart [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import 'package:scrolling/main.dart' as app;
+
+void main() {
+  // This line enables the extension.
+  enableFlutterDriverExtension();
+
+  // Call the `main()` function of the app or call `runApp` with
+  // any widget you are interested in testing.
+  app.main();
+}
```
Comment (domesticmouse) [inline null_safety_examples/cookbook/testing/integration/scrolling/test_driver/app.dart:12]: nit: trailing newline

File: null_safety_examples/cookbook/testing/integration/scrolling/test_driver/app_test.dart [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      );
+
+      // Verify that the item contains the correct text.
+      expect(
+        await driver.getText(itemFinder),
+        'Item 50',
+      );
+    });
+  });
+}
```
Comment (domesticmouse) [inline null_safety_examples/cookbook/testing/integration/scrolling/test_driver/app_test.dart:48]: ditto

**Review by domesticmouse — APPROVED** [review]
Ping me when you have fixed up the nits

**Comment by domesticmouse** [comment]
Actually I'm just going to land this as is. I now see that lack of trailing newlines on files is fairly common in this codebase,

**Comment by miquelbeltran** [comment]
Thanks! I will be sure to add trailing newlines for the next PRs

**Pull request merged**